src/main/groovy/org/pih/warehouse/inventory/CycleCountStartCommand.groovy

```diff
@@ -5,4 +5,17 @@ import grails.validation.Validateable
 class CycleCountStartCommand implements Validateable {
 
     CycleCountRequest cycleCountRequest
+
+    static constraints = {
+        cycleCountRequest(validator: { CycleCountRequest cycleCountRequest ->
+            CycleCount cycleCount = cycleCountRequest.cycleCount
+            if (!cycleCount) {
+                // When first starting a count, the cycle count object won't exist yet, so this is valid.
+                return true
+            }
```

Comment:
you don't need to check for valid cases, by default this would return `true`, so this check is unnecessary
Not a change request but if you were to remove this check, remember to add null safe operators or handle it somehow else below